packages/core/telemetry/admin/src/ampli/index.js

```diff
@@ -0,0 +1,380 @@
+/* tslint:disable */
+/* eslint-disable */
```

Comment:
if there is already a lint ignore generated why is it also in the .eslintignore ?